pkg/catalog/catalog.go

```diff
@@ -66,9 +65,94 @@ func Exists(client *occlient.Client, componentType string, componentVersion stri
 	return true, nil
 }
 
+// ListServices lists all the available service types
+func ListServices(client *occlient.Client) (ServiceTypeList, error) {
+
+	clusterServiceClasses, err := getClusterCatalogServices(client)
+	if err != nil {
+		return ServiceTypeList{}, errors.Wrapf(err, "unable to get cluster serviceClassExternalName")
```

Comment:
- Is there some typo in this error message?
- Also, the error message (in its current form) seems likely to confuse the CLI user instead of help them understand what failed.

Reply:
the error message is not meant for the cli user but for us.

Reply:
No typo, this is from previous code.